script_data/global/network/titles.yml

```diff
@@ -9,6 +9,7 @@ gui:
     - '&7* &eName: &7<[tagID]>'
     - '&7* &ePreview: &r<[tag].parsed>'
     - '&7* &eDescription: &7<[description]>'
+in_shop: true
```

Comment:
Per the issue opened with my instructions to fix this file; I failed to verify that this also matched an irrelevant entry here.
We just need to remove that part is all
```suggestion
```